components/CreateOrganizationForm.js

```diff
@@ -1,289 +1,387 @@
+                            {admins.map(admin => (
+                              <OnboardingProfileCard
+                                key={admin.member.id}
+                                collective={admin.member}
+                                adminCollective={LoggedInUser.collective}
+                                removeAdmin={removeAdmin}
+                              />
+                            ))}
+                          </Flex>
+                        )}
+                      </Flex>
+                      <Flex flexDirection="row" alignItems="center" justifyContent="space-around" mt={4}>
+                        <Flex fontSize="10px" mr={2}>
+                          <FormattedMessage id="inviteAdmin" defaultMessage="INVITE CO-ADMIN" />
+                        </Flex>
+                        <StyledHr flex="1" borderStyle="solid" borderColor="black.300" width={[100, 110, 120]} />
+                      </Flex>
+                      <CollectivePickerAsync
+                        creatable
+                        collective={null}
+                        types={['USER']}
+                        data-cy="admin-picker-org"
+                        value="pp"
+                        onChange={option => {
+                          const duplicates = admins.filter(admin => admin.member.id === option.value.id);
+                          setAdmins(duplicates.length ? admins : [...admins, { role: 'ADMIN', member: option.value }]);
+                        }}
+                        placeholder={intl.formatMessage(placeholders.username)}
+                      />
+                    </Container>
+                  </Flex>
+                </Container>
 
-            <div className="actions">
-              <StyledButton
-                buttonStyle="primary"
-                type="submit"
-                onClick={this.handleSubmit}
-                disabled={loading || !this.state.modified}
-              >
-                {loading ? (
-                  <FormattedMessage id="loading" defaultMessage="loading" />
-                ) : (
-                  <FormattedMessage id="organization.create" defaultMessage="Create organization" />
-                )}
-              </StyledButton>
-            </div>
-          </div>
-        </div>
-      </div>
-    );
-  }
+                <Flex flexDirection="column" my={4} mx={1} width={[320, 450]}>
+                  <StyledCheckbox
+                    name="authorization"
+                    required
+                    label={
+                      <FormattedMessage
+                        id="createorganization.authorization.label"
+                        defaultMessage="I verify that I am an authorized representative of this organization and 
+                            have the right to act on its behalf."
+                      />
+                    }
+                    onChange={({ checked }) => {
```

Comment:
If I check the authorization box but then uncheck it, and submit the form, it still submits the form. We shouldn't be able to submit the form if the box becomes unchecked.

Reply:
Done